saleor/graphql/translations/types.py

```diff
@@ -46,8 +46,9 @@ def get_translatable_attribute_values(attributes: list) -> List[AttributeValue]:
     for assignment in attributes:
         attr = assignment["attribute"]
         if attr.input_type in AttributeInputType.TRANSLATABLE_ATTRIBUTES:
-            value = assignment["values"][0]
-            translatable_values.append(value)
+            if assignment["values"]:
+                value = assignment["values"][0]
```

Comment:
I see we assume there will always be one value here. Why not extend the `translatable_values` list with `assignment["values"]` then? It would work for empty lists, single element lists, and would even continue to work if we were to allow multi-value fields.

Reply:
Done 👍